pages/search/index.tsx

```diff
@@ -6,11 +6,13 @@ import {
   SimpleGrid,
 } from "@nypl/design-system-react-components"
 import { useRouter } from "next/router"
+import { isEmpty as _isEmpty } from "underscore"
```

Comment:
I _think_ I started this convention so we'd know where these functions were coming from explicitly. It doesn't have to be this way now and I wonder if it is actually helpful or not. I'd be happy with whatever is chosen but just want to point out that we don't have to strictly stick to some conventions if they don't make sense anymore.

Reply:
Thanks! I figured that was the reason for the pattern, I think I'd prefer to use the default naming if there is no naming clash, but happy to keep the aliasing convention if anyone prefers it.